R/misc-checks.R

```diff
@@ -338,3 +338,23 @@ left_assign <- function (path) {
 
     return (assigns)
 }
+
+#' Check that the package contains no useless files like `.DS_Store`.
+#' @inheritParams pkg_has_no_scrap
+#' @return 'TRUE' if all files generated by 'roxygen2', otherwise 'FALSE'
+#' @noRd
+pkg_has_no_scrap <- function (path) {
+
+    all_contents <- gert::git_ls ()$path
```

Comment:
smart :ok_hand:

Reply:
Yeah, because main usage is actually intended to be local, so most people will actually have these scrap files